app/assets/javascripts/discourse/app/components/sidebar/admin-header.gjs

```diff
@@ -0,0 +1,28 @@
+import Component from "@glimmer/component";
+import { service } from "@ember/service";
+import { ADMIN_PANEL } from "discourse/lib/sidebar/panels";
+import BackToForum from "./back-to-forum";
+import Filter from "./filter";
+import ToggleAllSections from "./toggle-all-sections";
+
+export default class AdminHeader extends Component {
+  @service sidebarState;
+
+  get shouldDisplay() {
+    return this.sidebarState.currentPanel.key === ADMIN_PANEL;
+  }
+
+  <template>
+    {{#if this.shouldDisplay}}
+      <div class="admin-header">
```

Comment:
```suggestion
      <div class="sidebar-admin-header">
```

Minor nit but I think these classes should be a little more tightly scoped to sidebar